### PR TITLE
Unify the Cue configuration approach

### PR DESCRIPTION
**Pull request by bradbeck**
Flesh out the approach of using Cue for configuration using the Schema/Policy/Data hierarchy when the package is spread across multiple directory levels. e.g. Schema at the root, Policy in the intermediates sub-directories and Data in the leaf directories.

All of the stuff in `cue.mod/gen/...` is generated from the `cue get go ...` command importing go modules and converting them to cue. These can then be imported and used to verify the structure of things such as `ConfigMap`'s. `ssf.cue` imports several of these and creates base structures based on these imported go structs.

### CODE REVIEW

File: resources/kyverno/admission-control-policy/released/verify-attestation.json [hunk 1]
```diff
@@ -1,21 +1,11 @@
...
-        "validationFailureAction": "enforce",
-        "background": false,
-        "webhookTimeoutSeconds": 30,
-        "failurePolicy": "Fail",
         "rules": [
             {
+                "name": "attest-code-review",
                 "verifyImages": [
                     {
+                        "image": "ttl.sh/*",
```
Comment (trmiller) [inline resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: any way to tighten this up?

**Comment by trmiller** [comment]
Overall LGTM - just had the one minor question in here.

File: resources/kyverno/admission-control-policy/released/verify-attestation.json (hunk 1 above)
Comment (pxp928) [reply resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: This was done as some of the examples use a random name to store the image:
```
DEFAULT_REPOSITORY=$(xxd -l 16 -c 16 -p < /dev/random)
: "${REGISTRY:=ttl.sh}"
: "${REPOSITORY:=$REGISTRY/$DEFAULT_REPOSITORY}"
```

Comment (bradbeck) [reply resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: We could put a more specific image pattern here if we had a good idea of the use cases we want to support. There was some discussion yesterday about setting up different policies for different namespaces.

Comment (pxp928) [reply resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: I agree. This policy needs to be revised in a future PR but should not be a blocker for this. I created an issue to track this needed change #113

Comment (mlieberman85) [reply resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: I do think we should separate out example policies in some way. Whether that's other namespaces or what.

Comment (trmiller) [reply resources/kyverno/admission-control-policy/released/verify-attestation.json:8]: Cool - let's track it there - thanks

**Review by trmiller — APPROVED** [review]

**Review by mlieberman85 — APPROVED** [review]
LGTM

**Pull request merged**